_posts/en/newsletters/2019-09-04-newsletter.md

```diff
@@ -0,0 +1,131 @@
+---
+title: 'Bitcoin Optech Newsletter #62'
+permalink: /en/newsletters/2019/09/04/
+name: 2019-09-04-newsletter
+slug: 2019-09-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter relays a security announcement for LN
+implementations, describes a non-interactive coinjoin proposal, and
+notes a few changes in popular Bitcoin Infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade LN implementations:** one or more issues that are scheduled
+  to be disclosed at the end of this month affect older LN
+  implementations.  If you are using any of the following software
+  versions, upgrading to a more recent version is [strongly
+  recommended:][cve ln]
+
+    - C-Lightning < 0.7.1
+    - LND < 0.7
+    - Eclair <= 0.3
+
+## News
+
+- **SNICKER proposed:** Adam Gibson posted to the Bitcoin-Dev mailing
+  list with a [proposal][snicker] for *Simple Non-Interactive Coinjoin
+  with Keys for Encryption Reused* (SNICKER), a two-step method for
+  allowing wallets to create coinjoins non-interactively.  In the first
```

Comment:
suggestion: name the two steps (proposal/reception) before launching into the detail about each one